core/domain/activity_services.py

```diff
@@ -83,8 +110,16 @@ def split_by_type(activity_references):
     The elements in each of the returned lists are in the same order as those
     in the input list.
 
-    If any activity reference is incorrectly formatted, this raises an
-    Exception.
+    Args:
+        activity_references: A list of ActivityReference domain objects.
+    
+    Returns:
+        A two-tuple where first element of the tuple are exploration ids
```

Comment:
Some grammar issues ("first element are", missing "the"). Suggest:

A 2-tuple whose first element is a list of all exploration ids represented in the input list, and whose second element is a list of all collection ids represented in the input list.